src/core/config/transformer/rename_types.rs

```diff
@@ -65,6 +83,51 @@ impl Transform for RenameTypes {
                         }
                     }
                 }
+
+                // replace in interface.
+                type_.implements = type_
+                    .implements
+                    .iter()
+                    .filter_map(|interface_type_name| {
```

Comment:
Why are you using `filter_map` when you are always returning `Some`? 

Can a normal `map` with `match` work instead?

Reply:
yep, replace with map.